app/views/identifications/_identification_with_observation.html.erb

```diff
@@ -1,19 +0,0 @@
-<%- identification ||= identification_with_observation || @identification -%>
-<div class="mini <%= 'compact' if request.format.mobile? %> observations observationcol column span-8">
```

Comment:
I left out the compact style in the haml. I'm guessed that it's not required with bootstrap columns resizing?